components/channel_layout/channel_controller.jsx

```diff
@@ -88,6 +89,7 @@ export default class ChannelController extends React.Component {
                     <EditPostModal/>
                     <ResetStatusModal/>
                     <LeavePrivateChannelModal/>
+                    <SidebarWhatsNewModal/>
```

Comment:
Maybe we could use the `ModalController` for this modal since it only appears once?